package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@fastify/type-provider-typebox",
+  "version": "0.1.0-beta.0",
+  "description": "A Type Provider for Typebox over Fastify",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "dependencies": {
+    "@sinclair/typebox": "^0.23.0"
+  },
+  "peerDependencies": {
+    "fastify": "github:fastify/fastify#next"
+  },
+  "scripts": {
+    "build": "rimraf ./dist && mkdir dist && tsc --outDir dist",
+    "test": "npm run build && npm run typescript",
+    "typescript": "tsd"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fastify/fastify-type-provider-typebox.git"
+  },
+  "keywords": [
+    "typebox",
+    "fastify"
+  ],
+  "author": "RafaelGSS",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/fastify/fastify-type-provider-typebox/issues"
+  },
+  "homepage": "https://github.com/fastify/fastify-type-provider-typebox#readme",
+  "devDependencies": {
+    "fastify-tsconfig": "^1.0.1",
```

Comment:
@RafaelGSS Just picked up on a small thing. May want to install `@types/node` as a dev dependency such that `FastifyInstance` type can be initialized with the `http.Server` RawServer default. Currently the `req.headers` resolves to type any as TypeScript cannot locate the underlying headers object.

Reply:
Good catch!